includes/class-scanner.php

```diff
@@ -271,14 +272,64 @@ public static function notify_plugin_authors( $plugin, $results, $tag ) {
 		);
 
 		// $email->send();
-		// Temporarily don't email it to the author, just log it on WordPress.org instead.
-		if ( defined( 'PLUGIN_REVIEW_ALERT_SLACK_CHANNEL' ) && function_exists( 'slack_dm' ) ) {
-			$slack_body = $email->body();
-			// Convert the code blocks to slack code blocks.
-			$slack_body = preg_replace( '!((\n\d+[^\n]*)+)!sm', "\n```\$1\n```", $slack_body );
+	}
+
+	public static function notify_slack_channel( $plugin, $results, $tag ) {
+
+		$totals = sprintf(
+			"Found %d errors in %s %s.\n\n",
+			$results[ 'totals' ][ 'errors' ],
+			$plugin->post_name,
+			$tag
+		);
+
+		$summary = [];
+		foreach ( $results[ 'files' ] as $pathname => $file ) {
+			list( $slug, $filename ) = explode( '/', $pathname, 2 );
+			foreach ( $file[ 'messages' ] as $message ) {
+				// Skip warnings for now
+				if ( 'WARNING' === $message['type'] ) {
+					continue;
+				}
 
+				// Count the instances of each error per filename
+				$summary[ $message['source'] ][ $filename ][ $message['line'] ] = true;
+			}
+		}
+
+		if ( empty( $summary ) ) {
+			return;
+		}
+
+		// Get the order of magnitude of the number of installs
+		$active_installs = pow( 10, floor( log10( get_post_meta( $plugin->ID, 'active_installs', true ) ) ) );
```

Comment:
`active_installs` is already rounded to the nearest displayable value.

Reply:
Fixed in 9655992b79781160b71a80e5e3102b745a0272a7